core/arch/arm/plat-bcm/bcm_elog.c

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2019 Broadcom.
+ */
+
+#include <bcm_elog.h>
+#include <io.h>
+
+static struct bcm_elog global_elog;
+
+void bcm_elog_putchar(char ch)
+{
+	struct bcm_elog *elog = &global_elog;
+	uint32_t offset = 0, len = 0;
+	vaddr_t base = 0;
+
+	base = (vaddr_t)io_pa_or_va(&elog->base);
```

Comment:
Cast not needed

Reply:
Removed here and at other places as well